consumer/schemas/inventory.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Device(BaseModel):
+    id: int
+    serial: str
+    device_id: str
+    is_checkout: bool
+    model: Optional[str]
+    manufacturer: Optional[str]
+    location: Optional[str]
+
+    @classmethod
+    def serialize(cls, device: dict) -> "Device":
+        """Simplifies reuse across inventory API."""
+
+        def name_or_none(item: dict) -> Optional[str]:
```

Comment:
This method feels a bit redundant imho. `dict.get("name", None)` already returns None if not found. Even without `None`, i.e.g `dict.get("name")` already returns `None` by default.

For readability, I'd argue to replace the use of `name_or_none()` below with just the `dict.get("name")` method; unless I am missing something that gives an advantage here! 

i.e.: 
```python
model=device["model"].get("name"),
manufacturer=device["manufacturer"].get("name"),
location=device["location"]).get("name"),
```

Reply:
> For readability, I'd argue to replace the use of name_or_none() below with just the dict.get("name") method; unless I am missing something that gives an advantage here!

I agree, however, the issue was that the dictionary parameter may be `None` rather than `{}`. So `device["model"].get("name")` would throw an `Attribute Error`. [This was added in this commit here](https://github.com/ideafast/middleware-services/pull/13/commits/057f4dbc12088531e003b94ba071351fda3cd28f). You also cannot chain `.get` calls, e.g., `device.get("model").get("name")` although we could if the default is a dict `device.get("model", {}).get("name")`

Shall I update the type of the argument to optional and add a comment to explain this need?

Reply:
Ah I see, that makes sense. Happy to continue with `name_or_none` if you can add a quick comment for its use case 👍

Reply:
Went with the chaining option 🚀